app/models.py

```diff
@@ -1333,6 +1333,19 @@ class TemplateEmailFile(TemplateEmailFileBase):
 
     version = db.Column(db.Integer, default=0, nullable=False)
 
+    @classmethod
+    def from_json(cls, data):
+        """
+        Assumption: data has been validated appropriately.
+
+        Returns a Service object based on the provided data. Deserialises created_by to created_by_id as marshmallow
```

Comment:
the docstring hasa copypaste error - this method returns a `TemplateEmailFile` object, not a `Service object.`